odoo/openupgrade/doc/source/migration_details.rst

```diff
@@ -65,3 +94,107 @@ to a separate stanza in the server configuration file under a header
+
+    During handling of the above exception, another exception occurred:
+
+    Traceback (most recent call last):
+      File "/home/user/OpenUpgrade/odoo/service/server.py", line 925, in preload_registries
+        registry = Registry.new(dbname, update_module=update_module)
+      File "/home/user/OpenUpgrade/odoo/modules/registry.py", line 85, in new
+        odoo.modules.load_modules(registry._db, force_demo, status, update_module)
+      File "/home/user/OpenUpgrade/odoo/modules/loading.py", line 350, in load_modules
+        loaded_modules, processed_modules = load_module_graph(cr, graph, status, perform_checks=update_module, report=report, upg_registry=upg_registry)
+      File "/home/user/OpenUpgrade/odoo/modules/loading.py", line 139, in load_module_graph
+        migrations.migrate_module(package, 'pre')
+      File "/home/user/OpenUpgrade/odoo/modules/migration.py", line 174, in migrate_module
+        migrate(self.cr, installed_version)
+      File "/home/user/.virtualenvs/python3/lib/python3.6/site-packages/openupgradelib/openupgrade.py", line 1204, in wrapped_function
+        (module, filename, str(e).decode('utf8')))
+    AttributeError: 'str' object has no attribute 'decode'
+
+  You need to make sure to use the openupgradelib from github:
+
+  .. code-block:: bash
+
+    $ pip install --upgrade git+git://github.com/OCA/openupgradelib.git
+
+
+OpenUpgrade to version 11 called with python2
+---------------------------------------------
+
+  When the upgrade fails with a database initialization error like "load_module arg#2 should be a file or None"
+
+  .. code-block:: console
+
+    2018-06-13 07:47:18,143 6739 CRITICAL odoo_10_migrate_to_v11 odoo.service.server: Failed to initialize database `odoo_10_migrate_to_v11`.
+    Traceback (most recent call last):
+      File "/home/user/OpenUpgrade/odoo/service/server.py", line 925, in preload_registries
+        registry = Registry.new(dbname, update_module=update_module)
+      File "/home/user/OpenUpgrade/odoo/modules/registry.py", line 85, in new
+        odoo.modules.load_modules(registry._db, force_demo, status, update_module)
+      File "/home/user/OpenUpgrade/odoo/modules/loading.py", line 350, in load_modules
+        loaded_modules, processed_modules = load_module_graph(cr, graph, status, perform_checks=update_module, report=report, upg_registry=upg_registry)
+      File "/home/user/OpenUpgrade/odoo/modules/loading.py", line 139, in load_module_graph
+        migrations.migrate_module(package, 'pre')
+      File "/home/user/OpenUpgrade/odoo/modules/migration.py", line 165, in migrate_module
+        mod = load_script(pyfile, name)
+      File "/home/user/OpenUpgrade/odoo/modules/migration.py", line 26, in load_script
+        return imp.load_module(module_name, fp, fname, ('.py', 'r', imp.PY_SOURCE))
+    ValueError: load_module arg#2 should be a file or None
+
+  You need to make sure you are calling OpenUpgrade for upgrading to version 11 with python3:
```

Comment:
This is a requirement of v11, not OpenUpgrade itself. One should know this, but well we can remind it, but not with all this text.

Reply:
All this text should help users that strugle with using OpenUpgrade (and the goal was to improve the documentation so OpenUpgrade is easier to use) to find the solution to their problem with google. I could shorten the stacktrace to the first and last line with 3 dots in between. Would that help?

Reply:
The text can vary on time, so the stack trace is not useful at all. Just say that OpenUpgrade v11 needed Python 3, and not here in this section.

Reply:
If we remove the stack trace completely we would loose the ability to search for errors on google and find the correct solution.

Reply:
I repeat: this stack trace would change in the future, so it's useless (maybe it has changed already now that we merged latest changes in #1398)